src/LanguageServer/Protocol/Handler/Rename/PrepareRenameHandler.cs

```diff
@@ -33,7 +33,7 @@ public LSP.TextDocumentIdentifier GetTextDocumentIdentifier(LSP.PrepareRenamePar
         var position = await document.GetPositionFromLinePositionAsync(linePosition, cancellationToken).ConfigureAwait(false);
 
         var symbolicRenameInfo = await SymbolicRenameInfo.GetRenameInfoAsync(
-            document, position, cancellationToken).ConfigureAwait(false);
+            document, position, includeSourceGenerated: false, cancellationToken).ConfigureAwait(false);
```

Comment:
same question. basically, it's not apparent to me at callsites what teh right value is here.